draft-ietf-rats-corim.md

```diff
@@ -1552,58 +1558,38 @@ Attestation Results Set (ARS):
 
 ### Internal Representation of Conceptual Messages {#sec-ir-cm}
 
-Conceptual Messages are the inputs to a Verifier and may include Evidence, Reference Values, Endorsed Values, or Appraisal Policy.
-Internal representations of Conceptual Messages are defined by {{sec-ir-evidence}}, {{sec-ir-ref-val}}, and {{sec-ir-end-val}}.
-The internal representation of Conceptual Messages are constructed from a common building block structure called Environment-Claims Tuple (ECT).
-Additionally, ECTs define an internal representation of the ACS and ARS. See {{sec-ir-acs}} and {{sec-ir-ars}}.
-
-ECTs have six attributes:
-
-1. The environment.
-2. The properties of the environment.
-3. The authority.
-4. The name space.
-5. The Conceptual Message type.
-6. The profile.
-
-Environment (label 1):
-
-: Identifies the Target Environment. Environments are identified using instance, class, or group identifiers. Typically, composite Attester's are composed of components, each having an environment identifier.
-
-Properties (label 2):
+Conceptual Messages are Verifier input and output values such as Evidence, Reference Values, Endorsed Values, Appraisal Policy, and Attestation Results.
 
-: Properties of the Target Environment.
+The internal representation of Conceptual Messages, as well as the ACS {{sec-ir-acs}} and ARS {{sec-ir-ars}}, are constructed from a common building block structure called Environment-Claims Tuple (ECT).
 
-Authority (label 3):
+ECTs have five attributes:
 
-: Identifies the entity that issued the tuple. A certain type of key material by which the authority (and corresponding provenance) of the tuple can be determined, such as the public key of an asymmetric key pair that is associated with an authority's PKIX certificate.
+1. Environment : Identifies the Target Environment. Environments are identified using instance, class, or group identifiers. Environments may be composed of elements, each having an element identifier.
 
-Name Space (label 4):
+2. Elements : Identifies the set of elements contained within a Target Environment and their trustworthiness Claims.
 
-: Identifies the name space from which the tuple was created.
+3. Authority : Identifies the entity that issued the tuple. A certain type of key material by which the authority (and corresponding provenance) of the tuple can be determined, such as the public key of an asymmetric key pair that is associated with an authority's PKIX certificate.
 
-CM Type (label 5):
+4. Conceptual Message Type : Identifies the type of Conceptual Message that originated the tuple.
 
-: Identifies the type of Conceptual Message that originated the tuple.
+5. Profile : The profile that defines this tuple. If no profile is used, this attribute is omitted.
```

Comment:
Purely typographic comment: there are structural elements that can be used to assemble an enumeration/definition list, rather than doing it by hand.  E.g.:

```
{: vspace="0"}
Environment:
: Identifies ...

Elements:
: Identifies ...
```

or

```
{:my-enum: style="format %d."}

{: my-enum}
* Environment: Indicates ...
* Elements: Indicates ...
```

The advantage is that rendering in the supported output formats (TXT, HTML, PDF) is taken care of automatically - and, hopefully, also optimally.

Reply:
Fixed in commit https://github.com/ietf-rats-wg/draft-ietf-rats-corim/pull/268/commits/7a01ca689f4013783ba5138c0d80232214e3976b